docs/providers/aws/events/apigateway.md

```diff
@@ -535,6 +535,7 @@ functions:
               - Authorization
               - X-Api-Key
               - X-Amz-Security-Token
+            allowCredentials: true
```

Comment:
Do we need more documentation here (an own section)? Or is this sufficient as it's only a simple config parameter?

/cc @eahefnawy